src/config/config.ts

```diff
@@ -340,7 +340,7 @@ export function initializeServerConfiguration(rawConfiguration: any) {
             );
         } catch (err) {
             // ignore
-            console.log('Error parsing localStorage.frontendConfig');
+            console.log('Error parsing localStorage.frontendConfig:' + localStorage.frontendConfig);
```

Comment:
Useful update to debugging if "localStorage.frontendConfig" testing is not working.